From d75ab65cd40188c0e772cf0521da8d8206ef8652 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Thu, 30 Apr 2015 13:49:41 +0800
Subject: [PATCH 0109/5242] MLK-10783-7 doc: dt-binding: mxs-phy: add
 compatible string "fsl,imx6ul-usbphy"

commit  950726073f04c523b247d0d48c9f096236086aed from
https://source.codeaurora.org/external/imx/linux-imx.git

add compatible string "fsl,imx6ul-usbphy"

Signed-off-by: Peter Chen <peter.chen@freescale.com>
(cherry picked from commit 45583aa93ee7d3f8e930854603121f7facdcffb9)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../devicetree/bindings/phy/mxs-usb-phy.txt        |    1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/phy/mxs-usb-phy.txt b/Documentation/devicetree/bindings/phy/mxs-usb-phy.txt
index 21aad8e..98ebdc8 100644
--- a/Documentation/devicetree/bindings/phy/mxs-usb-phy.txt
+++ b/Documentation/devicetree/bindings/phy/mxs-usb-phy.txt
@@ -7,6 +7,7 @@ Required properties:
 	* "fsl,imx6sl-usbphy" for imx6sl
 	* "fsl,vf610-usbphy" for Vybrid vf610
 	* "fsl,imx6sx-usbphy" for imx6sx
+	* "fsl,imx6ul-usbphy" for imx6ul
   "fsl,imx23-usbphy" is still a fallback for other strings
 - reg: Should contain registers location and length
 - interrupts: Should contain phy interrupt
-- 
1.7.9.5

